examples/docs_snippets/docs_snippets/concepts/assets/explicit_asset_dependency.py

```diff
@@ -9,3 +10,27 @@ def upstream_asset():
 @asset(ins={"upstream": AssetIn("upstream_asset")})
 def downstream_asset(upstream):
     return upstream + [4]
+
+
+# end_string_example
+
+
+# start_explicit_asset_key_example
+
+
+@asset
```

Comment:
We can probably remove everything except lines 31-33 to keep the example shorter

Reply:
I removed the `upstream_asset` bit, but kept the remaining two assets. My thinking was that it'd be good to demonstrate both ways of providing asset keys - thoughts?